rhc.gemspec

```diff
@@ -30,7 +30,6 @@ Gem::Specification.new do |s|
     sep
   ].join("\n")
 
-  s.add_dependency              'net-ssh',      '>= 2.0.11', '<= 2.9.2'
   s.add_dependency              'net-scp',      '>= 1.1.2'
   s.add_dependency              'net-ssh-multi','>= 1.2.0'
   s.add_dependency              'archive-tar-minitar'
```

Comment:
I'm pretty sure in our meeting Dan said that locking EVERY version of EVERY dependency to a known good one would take a lot more effort than its worth.